pkg/api/defaults-scheduler.go

```diff
@@ -18,16 +18,16 @@ var defaultSchedulerConfig = map[string]string{
 func (cs *ContainerService) setSchedulerConfig() {
 	o := cs.Properties.OrchestratorProfile
 
-	// If no user-configurable scheduler config values exists, use the defaults
```

Comment:
Let's move these optimizations to a separate PR.

Thanks!

Reply:
Resolved and applied here: https://github.com/Azure/aks-engine/pull/1791